### PR TITLE
Changes based on feedback

### PR DESCRIPTION
**Pull request by SeanKilleen**
Got some good feedback (thank you, kind folks!) 

Therefore, making some edits.

### CODE REVIEW

File: _campaign-2019/sean-killeen.md [hunk 1]
```diff
@@ -23,14 +27,17 @@ I believe that if we're going to serve the various members of our community, we
...
+...and so on (I anticipate many more). By thinking about these different groups, considering their reactions, and confirming with people who identify with these personas or types, we can find the biggest impact, confirm the changes we make will have the intended effect, and communicate more clearly to various stakeholders. I want to us to know our community more fully.
 
 ### Impact Maps: A Technique to Drive the Foundation's Mission
 
-For these types of organizations, I've been heavily inspired by [Gojko Adzic's book on Impact Mapping](https://www.impactmapping.org/book.html). At the risk of serious over-simplifying, I will sum it up by saying that for me, impact mapping is a way to bring many of your stakeholders along to achieve your goals, though that's not quite the original intended purpose (I'd be happy to speak about this subject at length with anyone). 
+Everyone wants to start with things they'll do, or pet ideas that are based on their perpsective. I want to map our problem space and our stakeholders, so that we are conscious about what actions we're taking and what impact they're supposed to achieve.
```
Comment (khalidabuhakmeh) [inline _campaign-2019/sean-killeen.md]: `Everyone wants to start with things they'll do, or pet ideas that are based on their perpsective.`

This line can't be backed up and is more opinion.

Comment (khalidabuhakmeh) [reply _campaign-2019/sean-killeen.md]: Also `perspective` is misspelled.

File: _campaign-2019/sean-killeen.md [hunk 2]
```diff
@@ -6,13 +6,17 @@ image: https://avatars3.githubusercontent.com/u/2148318
 
 # .NET Foundation Campaign: Sean Killeen
 
-**Why I'm running, in short**: I love this community dearly, have grown up with it, and would jump at the chance to represent it and make it better. I hope that most who have interacted with me have found me to be helpful, collaborative, and passionate about advancing .NET's impact on our world.
+**Why I'm running, in short**: I bring an empathetic approach, a community-focused mindset, the tools to explore problem spaces and define impacts, and skills around teaching and building consensus. I also bring a ton of energy, and some specific ideas to start with. I will help the Foundation define its problem space and impacts, help the community see -- and shape -- where we want to go, and will position us to start on work that has the biggest impact possible for our ecosystem.
```
Comment (khalidabuhakmeh) [inline _campaign-2019/sean-killeen.md]: `and will position us to start on work that has the biggest impact possible for our ecosystem`

This is a confusing statement, maybe shorten it?

"and will position the .NET Foundation to make the biggest impact on our ecosystem."

**Comment by SeanKilleen** [comment]
@khalidabuhakmeh great suggestions. Thank you!

**Pull request merged**